src/Umbraco.Infrastructure/HostedServices/ReportSiteTask.cs

```diff
@@ -52,13 +52,22 @@ public override async Task PerformExecuteAsync(object state)
 
             try
             {
-                // Send data to LIVE telemetry
-                s_httpClient.BaseAddress = new Uri("https://telemetry.umbraco.com/");
+
+                if (s_httpClient.BaseAddress is null)
+                {
+                    // Send data to LIVE telemetry
+                    s_httpClient.BaseAddress = new Uri("https://telemetry.umbraco.com/");
 
 #if DEBUG
-                // Send data to DEBUG telemetry service
-                s_httpClient.BaseAddress = new Uri("https://telemetry.rainbowsrock.net/");
+                    // Send data to DEBUG telemetry service
+                    s_httpClient.BaseAddress = new Uri("https://telemetry.rainbowsrock.net/");
+
+                    // Set a low timeout - no need to use a larger default timeout for this POST request
+                    s_httpClient.Timeout = new TimeSpan(0, 0, 1);
```

Comment:
Looks like you are only setting the timeout now in "debug", whereas previously it was looking to set this for all requests.

Reply:
good spot..